docker-compose-prod.yaml

```diff
@@ -0,0 +1,27 @@
+version: '2'
```

Comment:
Have you looked at docker compose version 3? https://github.com/docker/docker.github.io/blob/master/compose/compose-file/index.md